nb-configuration.xml

```diff
@@ -20,5 +20,14 @@
         <com-junichi11-netbeans-changelf.use-global>false</com-junichi11-netbeans-changelf.use-global>
         <org-netbeans-modules-javascript2-requirejs.enabled>true</org-netbeans-modules-javascript2-requirejs.enabled>
         <netbeans.hint.jdkPlatform>JDK_1.8</netbeans.hint.jdkPlatform>
+        <netbeans.checkstyle.format>true</netbeans.checkstyle.format>
```

Comment:
Was necessary to introduce a project level formatting section? Why?

Reply:
Removed.